### PR TITLE
Update help.xml_Chinese

### PR DESCRIPTION
**Pull request by jindouyunz**
updated on 4/21/2019, version 2, Jindouyunz

### CODE REVIEW

**Comment by erciccione** [comment]
Pinging past translators for review: @uiharu-s @bitOMG

File: app/src/main/res/values-zh-rCN/help.xml [hunk 1]
```diff
@@ -206,66 +196,46 @@
...
-        <h2>Scan for Nodes</h2>
-        <p>Additionally, you can scan the network for nodes. Monerujo will start
-        scanning the network for Remote Nodes on port 18089. It begins by asking your
-        bookmarked nodes for other peers in the Monero P2P network and then continues
-        by asking those for their peers, and so on. If you have no bookmarked nodes
-        (or they don&apos;t tell us about their peers),
-        Monerujo will go straight to the Monero seed nodes hardcoded into Monero. The
-        scan stops when it finds 10 remote nodes in total.</p>
+        <h1>节点</h1>
+        <h2>该文较长</h2>
```
Comment (Lafudoci) [inline app/src/main/res/values-zh-rCN/help.xml]: Please check if this translation is accurate. TLDR means brief version of guide.

Comment (jindouyunz) [reply app/src/main/res/values-zh-rCN/help.xml]: Thank you

**Review by Lafudoci — APPROVED** [review]
Translation reviewed.

**Review by erciccione — APPROVED** [review]

**Review by m2049r — APPROVED** [review]

**Pull request merged**